dspace-spring-rest/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -306,9 +330,215 @@ public void findOneTestWrongUUID() throws Exception {
+
+        //2. Three public items that are readable by Anonymous with different subjects
+        Item embargoedItem1 = ItemBuilder.createItem(context, col1)
+                                         .withTitle("embargoed item 1")
+                                         .withIssueDate("2017-10-17")
+                                         .withAuthor("Smith, Donald").withAuthor("Doe, John")
+                                         .withSubject("ExtraEntry")
+                                         .withEmbargoPeriod("6 months")
+                                         .build();
+
+        //2. Three public items that are readable by Anonymous with different subjects
+        Item publicItem1 = ItemBuilder.createItem(context, col1)
+                                      .withTitle("Public item 1")
+                                      .withIssueDate("2017-10-17")
+                                      .withAuthor("Smith, Donald").withAuthor("Doe, John")
+                                      .withSubject("ExtraEntry")
+                                      .build();
+
+        context.restoreAuthSystemState();
+
+        String token1 = getAuthToken(admin.getEmail(), password);
+
+
+        getClient(token1).perform(get("/api/core/items/" + embargoedItem1.getID()))
+                         .andExpect(status().isOk());
+
+    }
+
+    @Test
+    public void privateAdminAccessTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        //** GIVEN **
+        //1. A community-collection structure with one parent community with sub-community and two collections.
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                                          .withName("Parent Community")
+                                          .build();
+        Community child1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
+                                           .withName("Sub Community")
+                                           .build();
+        Collection col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1").build();
+
+        //2. Three public items that are readable by Anonymous with different subjects
+        Item privateItem1 = ItemBuilder.createItem(context, col1)
+                                       .withTitle("private item 1")
+                                       .withIssueDate("2017-10-17")
+                                       .withAuthor("Smith, Donald").withAuthor("Doe, John")
+                                       .withSubject("ExtraEntry")
+                                       .makeUnDiscoverable()
+                                       .build();
+
+
+        context.restoreAuthSystemState();
+
+        String token1 = getAuthToken(admin.getEmail(), password);
+
+
+        getClient(token1).perform(get("/api/core/items/" + privateItem1.getID()))
```

Comment:
It would be good to illustrate the failure for an anonymous user attempting to access this item.

Reply:
Anonymous users are allowed to access "private/undiscoverable" items. I also think this is confusing but that's how it works in DSpace 6 at the moment.